src/main/java/de/uni_tuebingen/ub/nppm/db/SaveHelper.java

```diff
@@ -47,6 +50,57 @@ public static List<Map<String, String>> selectAttribute(String table, String att
         return results;
     }
 
+    public static List<Map> selectAttributeMap(String table, String attributeString, int id) throws Exception {
+        List<Map> results = new ArrayList<>();
+
+        try ( Session session = getSession()) {
+            session.getTransaction().begin();
```

Comment:
If you only select something from the database without modifying, there is no transaction needed

Reply:
done:  look at commit: improvement of select functions